test/integration/targets/yum_repository/tasks/main.yml

```diff
@@ -205,6 +205,11 @@
         name: "{{ yum_repository_test_repo.name }}"
         state: absent
 
+    - name: CLEANUP | Install GPG key
+      rpm_key:
+          state: present
+          key: "{{ yum_repository_epel.gpgkey }}"
+
```

Comment:
This task will need to be moved up before the `install {{ yum_repository_test_package }}` task so that the key is installed before package installation is performed. The `CLEANUP | ` prefix should be removed from the task name.

The task will also need to be conditional, since not all platforms running the test define the `yum_repository_epel` variable.

Reply:
Will submit a new PR - disabling GPG checks for test packages; that seems to do the trick.